frontend/components/Bridge/BridgeCompleted.tsx

```diff
@@ -25,37 +27,41 @@ export const BridgeCompleted = ({
   const { goto } = usePageState();
 
   return (
-    <CardFlex
-      bordered={false}
-      title={<CardTitle title="Bridge Completed" />}
-      extra={<GoToMainPageButton />}
-    >
-      <Result
-        status="success"
-        subTitle={completionMessage || 'Funds have been bridged successfully.'}
-        extra={[
-          <Flex
-            key="bridge-completed"
-            gap={24}
-            vertical
-            style={{ paddingTop: 8 }}
-          >
-            <BridgeTransferFlow
-              fromChain={fromChain}
-              toChain={toChain}
-              transfers={transfers}
-            />
-            <Button
-              onClick={() => goto(Pages.ManageWallet)}
-              size="large"
-              style={{ alignSelf: 'center' }}
-            >
-              See wallet balance
-            </Button>
-          </Flex>,
-        ]}
-        style={{ padding: '24px 0' }}
-      />
-    </CardFlex>
+    <Flex justify="center" style={{ marginTop: 40 }}>
+      <CardFlex
+        $noBorder
+        bordered={false}
+        style={{ width: CARD_WIDTH, padding: 8 }}
+      >
+        <Flex justify="space-between" align="center">
+          <Title3>Bridge Completed</Title3>
+          <GoToMainPageButton />
+        </Flex>
+        <Result
+          status="success"
+          subTitle={
+            completionMessage || 'Funds have been bridged successfully.'
+          }
+          extra={[
+            <Flex key="bridge-completed" gap={24} vertical className="pt-8">
+              <BridgeTransferFlow
+                fromChain={fromChain}
+                toChain={toChain}
+                transfers={transfers}
+                isBridgeCompleted
+              />
+              <Button
+                onClick={() => goto(Pages.ManageWallet)}
+                size="large"
+                style={{ alignSelf: 'center' }}
+              >
+                See wallet balance
+              </Button>
+            </Flex>,
+          ]}
+          style={{ padding: '24px 0' }}
```

Comment:
would be nice to avoid using so much "style", they can cause unnecessary re-renders. this can be handled by setting mb-24 (will need to add to globals.scss)